src/components/markdown-render/inlineCode.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Highlight, { defaultProps } from 'prism-react-renderer';
-import customTheme from 'prism-react-renderer/themes/github';
+import customTheme from 'prism-react-renderer/themes/vsDark';
```

Comment:
lets get this theme switch ran by design

Reply:
Design has signed off on this.